lib/rubocop/cop/rspec/leaked_const.rb

```diff
@@ -0,0 +1,61 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that constants are not created in tests without cleanup.
+      #
+      # Prefer stub_const and anonymous classes.
+      #
+      # @example
+      #   # bad
+      #   describe MyClass do
+      #     class Foo
+      #       BAR = 1
+      #     end
+      #   end
+      #
+      #   # good
+      #   describe MyClass do
+      #     before do
+      #       stub_const('Foo::BAR', 1)
+      #     end
+      #   end
+      class LeakedConst < Cop
+        MSG = 'Opening a class to define methods can pollute your tests. '\
```

Comment:
I agree about the message. I had a hard time coming up with good wording. This was the best I could do, but I'll keep trying to re-work it.

To your comment about using stub_const, maybe we shouldn't suggest one over the other in the message since people may have different preferences?